inc/back-compat.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
I assume this file need to be included / called in `functions.php` somewhere.

Reply:
Yes, of course it does. 🤦